packages/vike-react-query/integration/Wrapper.tsx

```diff
@@ -25,3 +29,24 @@ function Wrapper({ children }: { children: ReactNode }) {
 function PassThrough({ children }: any) {
   return <>{children}</>
 }
+
+function makeQueryClient(config: QueryClientConfig | undefined) {
+  return new QueryClient(config)
+}
+
+let clientQueryClient: QueryClient | undefined = undefined
+
+function getQueryClient(config: QueryClientConfig | undefined) {
+  if (isBrowser()) {
+    if (!clientQueryClient) clientQueryClient = makeQueryClient(config)
+    return clientQueryClient
+  } else {
+    return makeQueryClient(config)
+  }
+}
+
+function isBrowser() {
```

Comment:
copied from [vike-react-styled-components](https://github.com/vikejs/vike-react/blob/main/packages/vike-react-styled-components/Wrapper.tsx#L19). 
Is there a better way to share this code, or it would be unnecessary?

Reply:
I don't think it's worth it.